server/src/server.ts

```diff
@@ -1,16 +1,18 @@
 import {
-    createConnection,
-    TextDocuments,
-    ProposedFeatures,
-    InitializeParams,
-    DidChangeConfigurationNotification,
-    TextDocumentSyncKind,
-    InitializeResult
+	createConnection,
+	TextDocuments,
+	ProposedFeatures,
+	InitializeParams,
+	DidChangeConfigurationNotification,
+	TextDocumentSyncKind,
+	InitializeResult, DocumentLinkParams, DocumentLink
```

Comment:
Just like the other imports, can this all be on a new line?

Reply:
done in upcoming review fixes comm